### ISSUE TITLE
No module named 'tensorflow.google'

### ISSUE THREAD

**Issue by fangli08** [tensorflow/models#9745]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [x] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [x] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [x] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/blob/master/official/nlp/xlnet/preprocess_pretrain_data.py

## 2. Describe the bug

This bug is simply that the `tensorflow.google` module can not be found. this module is used in the entire script so it is essential that it should be fixed. 
```
Traceback (most recent call last):
  File "preprocess_pretrain_data.py", line 33, in <module>
    import tensorflow.google as tf
ModuleNotFoundError: No module named 'tensorflow.google'
```

## 3. Steps to reproduce

Steps to reproduce the behavior.

run this command: `python preprocess_pretrain_data.py` 
my tensorflow version is 2.4.1
## 4. Expected behavior

I guess 'tensorflow.google' is an internal module that is used inside google? 

## 5. Additional context

Include any logs that would be helpful to diagnose the problem.

## 6. System information

- OS Platform and Distribution:Linux Ubuntu 18.04
- TensorFlow installed from: pip install tensorflow
- TensorFlow version: v2.4.0-49-g85c8b2a817f   2.4.1
- Python version: 3.6.12
- Bazel version (if compiling from source): NA
- GCC/Compiler version (if compiling from source): NA
- CUDA/cuDNN version: 11.1
- GPU model and memory: GTX 1080 Ti, 11GB

**Comment by saberkun** [comment]
Ooops. Sorry about that. it should import tensorflow.compat.v1 as tf.
The code may not maintain well. Will fix and check if the processing code is still correct.
@allenwang28

**Comment by allenwang28** [comment]
Thanks for raising this issue, it should be fixed in this commit: https://github.com/tensorflow/models/commit/89031e1afa718861ae40048c117501000d8852d6.

I will note that we have had some difficulties in verifying pretrain convergence for XLNet which is currently an ongoing effort. This preprocessing script is probably close to the original version, but we've created a fork of this [here](https://github.com/tensorflow/models/blob/master/official/nlp/data/create_xlnet_pretraining_data.py) that is also tested and will be maintained moving forward.

**Comment by fangli08** [comment]
Thanks! @allenwang28 and @saberkun

**Comment by fangli08** [comment]
@allenwang28 
>I will note that we have had some difficulties in verifying pretrain convergence for XLNet which is currently an ongoing effort.

I observed the same thing using the orginical version of XLNet

I initialize parameters from the released xlnet-base and pretrain it on the biomedical corpora.  
If I use the GPU version (`train_gpu.py`), it converges well and the loss is approaching to 1.35 after 80k steps. The loss are within 1.33-1.38 (logging step = 1000, batch size = 16), which I think is reasonable. However, for the TPU version (`train.py`), the loss varies between 0.9-1.6 (same batch size, input tfrecords and start from the same checkpoint), even if I set the learning rate to 1e-10. My input text is shuffled so there shouldn't be such big difference. I hope this information would help you identify the issue.
